### DISCUSSION TITLE
Problem with faceswap

### DISCUSSION THREAD

**Discussion by AndreiDan93** [lllyasviel/Fooocus#1506] [Q&A]
Hi, when I generate a person with fooocus faceswap function, it changes the face almost every time. I want to have a person, that has the same face every time even after I close and reopen the software. Can someone help me please? Thanks in advance!

**Comment by IPv6** [comment]
There is no such direct functionality. You can try fixed seeds, etc, but a lot of experimentations, inpainting will be needed anyway

**Reply by AndreiDan93** [reply]
This guy did an entire instagram page.

https://youtu.be/ky5ZB-mqZKM?si=ejnQydlKbZD3QhA-

**Reply by IPv6** [reply]
Well, i think he is good with photoshop, experimenting and manual work :)

**Reply by sheriffdude** [reply]
hi guys where do i find the face swap cant see it using foocus 2.0.78.5 mre have i missed something great program

**Reply by Bratzmeister** [reply]
@sheriffdude first off I think you should update to a more recent version. you can find the face swap when you enable image prompt and check the "advanced" checkbox below that lets you choose from the currently 4 different img2img methods implemented in fooocus. you can also set weights there

**Comment by Hypersonic-Popcorn** [comment]
You can try using one or more names, usually celebrities, but you're probably going to get their face. Multiple names might mix them a bit more but it's still going to look a lot like them. Using a name of a different gender may look a bit less recognizable.
Also, loras and embeddings do this well. On Civitai, search for "nobodies" and you should find a bunch. Most of them are for SD1.5 but there are a few collections for SDXL.

**Reply by AndreiDan93** [reply]
I want to try and create a person and generate photos of that person only. ( a person that doesn't exist)

**Reply by Bratzmeister** [reply]
@AndreiDan93 you need to create and edit pictures of said person then train a LoRa on them. You can then use this LoRa to create more of said person. However that requires a lot of "training material" aka pictures to feed in the training so that you get good results.

**Reply by AndreiDan93** [reply]
That's what i'm trying now, it's hard to start with many pictures to train your AI if you can't create  the same person to have the photos.

**Comment by juliux32** [comment]
I'm kinda new to all this AI generation thing, but as far as I checked, googled, and youtubed, you will have to **create your own Lora/model**. There are a lot of Celebrity Loras created and for example, if you would use Lora of **Megan Fox** and type let's say **"person sitting on the floor"**, that person would look like **Megan Fox**.

**If I am wrong, please someone correct me and show of how you can do it in an easier way.**

**Reply by mashb1t** [reply]
@AndreiDan93 LoRA training can help you achieve the goal of getting consistent images resembling a specific subject, e.g. a person.
You can find tutorials in the kohya_ss repository, which is a tool that offers a very high amount of customizability:
https://github.com/bmaltais/kohya_ss?tab=readme-ov-file#tutorials

**Comment by sheriffdude** [comment]
hi guys why cant i find the box for image promt to face swap has it been removed from foocus 2.078.5 MRE
